src/pages/PageHome.jsx

```
@@ -33,26 +33,26 @@ export default function PageHome() {
           <option value="top_rated">Top Rated</option>
           <option value="upcoming">Upcoming</option>
         </select>
-        <ul>
+        <ul >
           {data?.results &&
             data.results.map((movieDetails) => (
               <MovieCard key={movieDetails.id} details={movieDetails} />
             ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)